### DISCUSSION TITLE
render_table missing id and name parameters

### DISCUSSION THREAD

**Discussion by sjoerdkt** [helloflask/bootstrap-flask#299] [Q&A]
Hello,

I want to target the rendered table using jquery becuase I would like to add a dblclick event on it (well the rows) but there is afaik no parameter to give the table an id. I checked the base template and I did not find a table_id (or id). Any idea's how to do that or did I found. I did not check further but imho all elements should be able to have an id

Regards
Sjoerd

**Comment by PanderMusubi** [comment]
If you want, you can make a PR to accommodate this.
